docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,14 @@
 title: Changelog
 ---
 
+## 13.13.0
+
+_Released 7/02/2024 (PENDING)_
+
+**Features:**
+
+- `.type({upArrow})` and `.type({downArrow})` now also works for date, month, week, time, datetime-local and range input types. Addresses [#29665](https://github.com/cypress-io/cypress/issues/29665).
+
```

Comment:
- There should not be any text added to the Changelog here. That is done as a bulk update as part of the release process which would pull in the change added in PR https://github.com/cypress-io/cypress/pull/29678

Reply:
Got it. Will do